src/library/scala/collection/LazyZipOps.scala

```diff
@@ -2,7 +2,13 @@ package scala.collection
 
 import scala.language.implicitConversions
 
-/** Decorator representing lazily zipped pairs. */
+/** Decorator representing lazily zipped pairs.
+  *
+  * @define coll pair
+  * @define willNotTerminateInf
```

Comment:
Both appear to be unused at the moment. We will need `willNotTerminateInf` but I think `coll` is unnecessary in these LazyZip classes (because they don't extend any standard collection traits, just like `ArrayOps` and `StringOps`)

Reply:
The `coll` here is used for the `IterableOnce` methods which are included via implicit conversion to `View`. Without these, we get "view" in the docs, but also a number of extra warnings when building the docs.